From b73efe72438710d7b2a7a08e22c9f1fd4d2907e7 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Thu, 12 Apr 2012 14:47:19 +0800
Subject: ENGR00179498-1 SDMA: fix p2p sdma script error

The p2p script in SDMA binary file is invalid. The ESAI call ASRC
can't work properly with this firmware.

Update the firmware and script address.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 firmware/imx/sdma/sdma-imx6q-to1.bin.ihex |  187 +++++++++++++++--------------
 1 file changed, 94 insertions(+), 93 deletions(-)

diff --git a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
index f9ef594..b969928 100644
--- a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
+++ b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
@@ -1,99 +1,100 @@
 :1000000053444D4101000000010000001C000000AD
-:1000100023000000A8000000780500008202000014
+:1000100025000000B000000078050000820200000A
 :10002000FFFFFFFF00000000FFFFFFFFFFFFFFFFDC
 :10003000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD0
-:10004000FFFFFFFFB61800006A1A00004B04000013
-:10005000EB0200004A190000FF180000080400002D
-:1000600095040000C0030000C105000070050000F9
-:1000700009040000AB020000E30400007B03000061
+:10004000FFFFFFFFFFFFFFFF6A1A0000FFFFFFFF38
+:10005000EB020000BB180000FFFFFFFF08040000D8
+:10006000FFFFFFFFC0030000FFFFFFFFFFFFFFFFD9
+:10007000FFFFFFFFAB020000FFFFFFFF7B0300005D
 :10008000FFFFFFFFFFFFFFFF4C0400006E040000B6
 :10009000FFFFFFFF00180000FFFFFFFFFFFFFFFF54
-:1000A0000000000000180000E3C1DB57E35FE357E6
-:1000B000F352016A8F00D500017D8D00A005EB5D34
-:1000C0007804037D79042C7D367C79041F7CEE5600
-:1000D000000F6006057D0965437E0A62417E209817
-:1000E0000A623E7E09653C7E12051205AD0260077C
-:1000F000037DFB55D36D2B98FB55041DD36DC86A4A
-:100100002F7F011F03200048E47C5398FB55D76DD7
-:10011000150005780962C86A0962C86AD76D5298E5
-:10012000FB55D76D1500150005780A62C86A0A628A
-:10013000C86AD76D5298FB55D76D1500150015008C
-:1001400005780B62C86A0B62C86AD76D097CDF6DDF
-:10015000077F0000EB55004D077DFAC1E357069875
-:100160000700CC680C6813C20AC20398D9C1E3C166
-:10017000DB57E35FE357F352216A8F00D500017D1F
-:100180008D00A005EB5DFB567804037D79042A7D84
-:10019000317C7904207C700B1103EB53000F60035A
-:1001A000057D0965377E0A62357E86980A62327E51
-:1001B0000965307E12051205AD026007027C065A01
-:1001C0008E98265A277F011F03200048E87C700B79
-:1001D00011031353AF98150004780962065A096297
-:1001E000265AAE981500150004780A62065A0A626B
-:1001F000265AAE9815001500150004780B62065AB1
-:100200000B62265A077C0000EB55004D067DFAC1B3
-:10021000E357699807000C6813C20AC26698700B0E
-:10022000110313536C07017CD9C1FB5E8A066B076F
-:10023000017CD9C1F35EDB59D3588F0110010F390E
-:100240008B003CC12B7DC05AC85B4EC1277C880304
-:100250008906E35CFF0D1105FF1DBC053E07004D3F
-:10026000187D700811007E07097D7D07027D2852E8
-:10027000E698F852DB54BC02CC02097C7C07027D74
-:100280002852EF98F852D354BC02CC02097D0004E6
-:10029000DD988B00C052C85359C1D67D0002CD985D
-:1002A000FF08BF007F07157D8804D500017D8D0004
-:1002B000A005EB5D8F0212021202FF3ADA05027C02
-:1002C0003E071899A402DD02027D3E0718995E07D9
-:1002D0001899EB559805EB5DF352FB546A07267DA0
-:1002E0006C07017D55996B07577C6907047D68078A
-:1002F000027D010E2F999358D600017D8E009355F3
-:10030000A005935DA00602780255045D1D7C004E99
-:10031000087C6907037D0255177E3C99045D147FB4
-:10032000890693500048017D2799A0991500067809
-:100330000255045D4F070255245D2F07017CA099EB
-:1003400017006F07017C012093559D000700A7D976
-:10035000F598D36C6907047D6807027D010E6499E6
-:100360009358D600017D8E009355A005935DA0069D
-:1003700002780255C86D0F7C004E087C6907037D2A
-:100380000255097E7199C86D067F89069350004811
-:10039000017D5C99A0999A99C36A6907047D6807F1
-:1003A000027D010E87999358D600017D8E009355EA
-:1003B000A005935DA0060278C865045D0F7C004E21
-:1003C000087C6907037DC865097E9499045D067FF2
-:1003D000890693500048017D7F99A09993559D000F
-:1003E0000700FF6CA7D9F5980000E354EB55004DCA
-:1003F000017CF598DD98E354EB55FF0A1102FF1AD2
-:100400007F07027CA005B4999D008C05BA05A00564
-:100410001002BA04AD0454040600E3C1DB57FB52DA
-:10042000C36AF352056A8F00D500017D8D00A005D7
-:10043000EB5D7804037D79042B7D1E7C7904337C8D
-:10044000EE56000FFB556007027DC36DD599041D64
-:10045000C36DC8623B7E6006027D10021202096A0B
-:10046000357F1202096A327F1202096A2F7F011F4B
-:1004700003200048E77C099AFB55C76D150015005D
-:1004800015000578C8620B6AC8620B6AC76D089AC6
-:10049000FB55C76D150015000578C8620A6AC86269
-:1004A0000A6AC76D089AFB55C76D15000578C862C2
-:1004B000096AC862096AC76D097C286A077F00005B
-:1004C000EB55004D057DFAC1DB57BF9977C2540447
-:1004D0000AC2BA99D9C1E3C1DB57F352056A8F004A
-:1004E000D500017D8D00A005FB567804037D7904BD
-:1004F000297D1F7C79042E7CE35D700D1105ED557F
-:10050000000F6007027D0652329A2652337E600544
-:10051000027D10021202096A2D7F1202096A2A7FE7
-:100520001202096A277F011F03200048EA7CE35575
-:100530005D9A150015001500047806520B6A2652C4
-:100540000B6A5C9A15001500047806520A6A265256
-:100550000A6A5C9A150004780652096A2652096AEA
-:10056000097C286A077F0000DB57004D057DFAC132
-:10057000DB571B9A77C254040AC2189AE3C1DB57AF
-:10058000F352056AFB568E02941AC36AC862690266
-:10059000247D941EC36ED36EC8624802C86A942636
-:1005A000981EC36ED36EC8624C02C86A9826C36E8A
-:1005B000981EC36EC8629826C36E6002097CC8622A
-:1005C0006E02247D096A1E7F0125004D257D849AD7
-:1005D000286A187F04627AC2B89AE36E8F00D80541
-:1005E000017D8D00A005C8626E02107D096A0A7F38
-:1005F0000120F97C286A067F0000004D0D7DFAC1BC
-:10060000DB576E9A070004620C6AB59A286AFA7F73
-:1006100004627AC258045404286AF47F0AC26B9AAE
+:1000A000000000000018000062180000161A00008E
+:1000B000E3C1DB57E35FE357F352016A8F00D500DA
+:1000C000017D8D00A005EB5D7804037D79042C7D16
+:1000D000367C79041F7CEE56000F6006057D0965AD
+:1000E000437E0A62417E20980A623E7E09653C7E1C
+:1000F00012051205AD026007037DFB55D36D2B98E9
+:10010000FB55041DD36DC86A2F7F011F03200048D3
+:10011000E47C5398FB55D76D150005780962C86AD1
+:100120000962C86AD76D5298FB55D76D1500150046
+:1001300005780A62C86A0A62C86AD76D5298FB5588
+:10014000D76D15001500150005780B62C86A0B62A3
+:10015000C86AD76D097CDF6D077F0000EB55004D45
+:10016000077DFAC1E35706980700CC680C6813C2F4
+:100170000AC20398D9C1E3C1DB57E35FE357F352E7
+:10018000216A8F00D500017D8D00A005EB5DFB5637
+:100190007804037D79042A7D317C7904207C700BFE
+:1001A0001103EB53000F6003057D0965377E0A627A
+:1001B000357E86980A62327E0965307E1205120508
+:1001C000AD026007027C065A8E98265A277F011FCF
+:1001D00003200048E87C700B11031353AF981500FF
+:1001E00004780962065A0962265AAE98150015006D
+:1001F00004780A62065A0A62265AAE98150015005B
+:10020000150004780B62065A0B62265A077C000020
+:10021000EB55004D067DFAC1E357699807000C685D
+:1002200013C20AC26698700B110313536C07017C4A
+:10023000D9C1FB5E8A066B07017CD9C1F35EDB592D
+:10024000D3588F0110010F398B003CC12B7DC05A50
+:10025000C85B4EC1277C88038906E35CFF0D11054E
+:10026000FF1DBC053E07004D187D700811007E077C
+:10027000097D7D07027D2852E698F852DB54BC02C6
+:10028000CC02097C7C07027D2852EF98F852D354A7
+:10029000BC02CC02097D0004DD988B00C052C8531B
+:1002A00059C1D67D0002CD98FF08BF007F07157D9C
+:1002B0008804D500017D8D00A005EB5D8F02120240
+:1002C0001202FF3ADA05027C3E071899A402DD0209
+:1002D000027D3E0718995E071899EB559805EB5D6E
+:1002E000F352FB546A07267D6C07017D55996B0715
+:1002F000577C6907047D6807027D010E2F9993588A
+:10030000D600017D8E009355A005935DA00602786E
+:100310000255045D1D7C004E087C6907037D025573
+:10032000177E3C99045D147F890693500048017D37
+:100330002799A099150006780255045D4F070255CC
+:10034000245D2F07017CA09917006F07017C012015
+:1003500093559D000700A7D9F598D36C6907047DD4
+:100360006807027D010E64999358D600017D8E00C6
+:100370009355A005935DA00602780255C86D0F7CC9
+:10038000004E087C6907037D0255097E7199C86D8E
+:10039000067F890693500048017D5C99A0999A993F
+:1003A000C36A6907047D6807027D010E8799935827
+:1003B000D600017D8E009355A005935DA0060278BE
+:1003C000C865045D0F7C004E087C6907037DC86525
+:1003D000097E9499045D067F890693500048017D4B
+:1003E0007F99A09993559D000700FF6CA7D9F598B8
+:1003F0000000E354EB55004D017CF598DD98E35483
+:10040000EB55FF0A1102FF1A7F07027CA005B49981
+:100410009D008C05BA05A0051002BA04AD04540471
+:100420000600E3C1DB57FB52C36AF352056A8F0033
+:10043000D500017D8D00A005EB5D7804037D790476
+:100440002B7D1E7C7904337CEE56000FFB55600734
+:10045000027DC36DD599041DC36DC8623B7E6006E5
+:10046000027D10021202096A357F1202096A327F88
+:100470001202096A2F7F011F03200048E77C099AB6
+:10048000FB55C76D1500150015000578C8620B6A8D
+:10049000C8620B6AC76D089AFB55C76D1500150039
+:1004A0000578C8620A6AC8620A6AC76D089AFB556D
+:1004B000C76D15000578C862096AC862096AC76D08
+:1004C000097C286A077F0000EB55004D057DFAC1C5
+:1004D000DB57BF9977C254040AC2BA99D9C1E3C1A4
+:1004E000DB57F352056A8F00D500017D8D00A00512
+:1004F000FB567804037D7904297D1F7C79042E7CCA
+:10050000E35D700D1105ED55000F6007027D065289
+:10051000329A2652337E6005027D10021202096A69
+:100520002D7F1202096A2A7F1202096A277F011FA2
+:1005300003200048EA7CE3555D9A1500150015007C
+:10054000047806520B6A26520B6A5C9A1500150055
+:10055000047806520A6A26520A6A5C9A15000478E0
+:100560000652096A2652096A097C286A077F000038
+:10057000DB57004D057DFAC1DB571B9A77C2540447
+:100580000AC2189AE3C1DB57F352056AFB568E0282
+:10059000941AC36AC8626902247D941EC36ED36E26
+:1005A000C8624802C86A9426981EC36ED36EC86299
+:1005B0004C02C86A9826C36E981EC36EC8629826FD
+:1005C000C36E6002097CC8626E02247D096A1E7FC8
+:1005D0000125004D257D849A286A187F04627AC21D
+:1005E000B89AE36E8F00D805017D8D00A005C86222
+:1005F0006E02107D096A0A7F0120F97C286A067F55
+:100600000000004D0D7DFAC1DB576E9A07000462B1
+:100610000C6AB59A286AFA7F04627AC258045404B4
+:08062000286AF47F0AC26B9AFC
 :00000001FF
-- 
1.7.9.5

